<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QDialog" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Setting</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QPushButton" name="exit">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="icon.qrc">
       <normaloff>:/resource/exit.svg</normaloff>:/resource/exit.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>40</width>
       <height>40</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>40</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>40</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_0">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_0">
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="S">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/resource/server.svg</normaloff>:/resource/server.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="server_port">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>JetBrains Mono</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>22</number>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_1">
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="U">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/resource/user.svg</normaloff>:/resource/user.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="username">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>JetBrains Mono</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>8</number>
           </property>
           <property name="placeholderText">
            <string extracomment="username"/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="rightMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="P">
           <property name="font">
            <font>
             <family>JetBrains Mono</family>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="icon.qrc">
             <normaloff>:/resource/passwd.svg</normaloff>:/resource/passwd.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="password">
           <property name="minimumSize">
            <size>
             <width>175</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>175</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>JetBrains Mono</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>12</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="placeholderText">
            <string comment="password" extracomment="password"/>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="cancel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>JetBrains Mono</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:10px;
background-color: rgb(188, 250, 109);</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="creteNewUser">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>JetBrains Mono</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:10px;
background-color: rgb(188, 250, 109);</string>
           </property>
           <property name="text">
            <string>Create</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Login">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>32</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>JetBrains Mono</family>
             <pointsize>8</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:10px;
background-color: rgb(188, 250, 109);</string>
           </property>
           <property name="text">
            <string>Login</string>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(32, 74, 135);</string>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <item>
          <widget class="QLabel" name="transparentLabel">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>透明度 </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="transparent">
           <property name="maximum">
            <number>255</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
           <property name="pageStep">
            <number>25</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
           <property name="sliderPosition">
            <number>100</number>
           </property>
           <property name="tracking">
            <bool>false</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="fontSizeLabel">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>字体大小</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="fontSizeSlider">
           <property name="minimum">
            <number>9</number>
           </property>
           <property name="maximum">
            <number>16</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>11</number>
           </property>
           <property name="sliderPosition">
            <number>11</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="mainWinWidthLabel">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
             <pointsize>9</pointsize>
            </font>
           </property>
           <property name="text">
            <string>主界面宽</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="mainWinWidthSlider">
           <property name="minimum">
            <number>280</number>
           </property>
           <property name="maximum">
            <number>560</number>
           </property>
           <property name="value">
            <number>300</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="chooseFontColor">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 楷体 Std R</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:3px;
</string>
           </property>
           <property name="text">
            <string>字体颜色</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="chooseIconColor">
           <property name="minimumSize">
            <size>
             <width>75</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>75</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Adobe 楷体 Std R</family>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius:3px;</string>
           </property>
           <property name="text">
            <string>图标颜色</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>40</number>
         </property>
         <property name="rightMargin">
          <number>80</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Adobe 黑体 Std R</family>
            </font>
           </property>
           <property name="text">
            <string>开机启动</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
